agent/transformers/influx_db.go

```diff
@@ -0,0 +1,126 @@
+package transformers
+
+import (
+	"errors"
+	"strconv"
+	"strings"
+
+	"github.com/sensu/sensu-go/types"
+)
+
+// InfluxList contains a list of Influx values
+type InfluxList []Influx
+
+// Influx contains values of influx db line metric format
+type Influx struct {
+	Measurement string
+	TagSet      []Tag
+	FieldSet    []Field
+	Timestamp   int64
+}
+
+// Tag is a key value pair representing an influxdb tag set
+type Tag struct {
+	Key   string
+	Value string
+}
+
+// Field is a key value pair representing an influxdb field set
+type Field struct {
```

Comment:
Same as `Tag`

Reply:
Yeah, we could probably move this to a `transformers.go` file, I like that idea.